packages/react-components/src/utils/setupConfiguration.ts

```diff
@@ -289,19 +311,60 @@ async function fetchVersionName(defaultValue: string): Promise<string> {
     return getConfigurationStanzaValue(service, 'app', 'launcher', 'version', defaultValue);
 }
 
+function convertSeverityFilterToArray(
+    severitiesFilter: string[],
+    availableSeverities: Severity[]
```

Comment:
I wonder if it makes sense for this to be called `allSeverities`. I'm OK with `availableSeverities` as well.